pkg/cloud/aws/actuators/scope.go

```diff
@@ -34,6 +34,8 @@ import (
 	"sigs.k8s.io/controller-runtime/pkg/patch"
 )
 
+const apiEndpointPort = 443
```

Comment:
Is this the only place we specify this port? I remember we talked about not hardcoding the port upstream, but I'm not sure if we made that configurable yet